pkg/operator/ceph/nfs/config.go

```diff
@@ -113,7 +113,7 @@ EXPORT_DEFAULTS {
 
 NFSv4 {
 	Delegations = false;
-	RecoveryBackend = 'rados_cluster';
+	RecoveryBackend = "rados_cluster";
```

Comment:
@BlaineEXE same change is required at https://github.com/rook/rook/blob/db23aa25fc3f6e5981c5d175f26f24a6f0e9ab3b/pkg/operator/ceph/nfs/config.go#L131 to workaround the problem.

Reply:
Thanks, missed that one